db/migrate/20170403153216_create_links.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLinks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :links do |t|
+      t.string :uri
```

Comment:
@kevindew I'm not sure if this restricts the length of the uri to a db-specific limit. Would `t.text` be better?